lib/src/component/mod.rs

```diff
@@ -12,6 +12,14 @@ component::bindgen!({
         "wasi:io": wasmtime_wasi::bindings::io,
         "wasi:cli": wasmtime_wasi::bindings::cli,
     },
+
+    trappable_error_type: {
+        "fastly:api/types/error" => types::TrappableError,
+    },
+
+    trappable_imports: [
+        "header-values-get"
+    ],
```

Comment:
It's a little funky that these types will be different even in the non-trapping config, but I think this probably the lightest touch possible for making the two modes work out (vs duplicating the macro invocation entirely between both configs... ugh)

Reply:
It also makes it so that we don't have to deal with nested `Result` values in the return type for `header-values-get`, which is much easier for me to think about :sweat_smile: